rosidl_generator_c/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ add_library(${PROJECT_NAME} SHARED
 )
 if(NOT WIN32)
   set_target_properties(${PROJECT_NAME} PROPERTIES
-    COMPILE_FLAGS "-std=c11 -Wall -Wextra")
+    COMPILE_FLAGS "-Wall -Wextra")
```

Comment:
I think we should definitely make use of the new features of C11. Therefore I would propose to keep this compile flag *in* in every package containing C code.